baby_squeel.gemspec

```
@@ -19,7 +19,7 @@ Gem::Specification.new do |spec|
 
   spec.files = Dir.glob('{lib/**/*,*.{md,txt,gemspec}}')
 
-  spec.add_dependency 'activerecord', '>= 6.1.5', '< 7.2'
+  spec.add_dependency 'activerecord', '>= 6.1.5', '< 8'
```

```suggestion
  spec.add_dependency 'activerecord', '>= 7.2', '< 8'
```
only compatible with 7.2